Classes/Accounts/List/Screens/HomeListLayout.swift

```diff
@@ -87,20 +87,15 @@ extension HomeListLayout {
                 sizeForAccountItem: item,
                 atSection: indexPath.section
             )
-        case .accountNotBackedUpWarning(let item):
-            return listView(
-                collectionView,
-                layout: collectionViewLayout,
-                sizeForAccountNotBackedUpWarningCellItem: item,
-                atSection: indexPath.section
-            )
         case .announcement(let item):
             return listView(
                 collectionView,
                 layout: collectionViewLayout,
                 sizeForAnnouncementCellItem: item,
                 atSection: indexPath.section
             )
+        case .carouselBanner:
+            return CGSize((collectionView.bounds.width, 90))
```

Comment:
Using tuples in `CGSize` construction is the part of the Macaroon framework. You should use standard initialiser If you want to loose our dependency with this framework.